tests/integration/cv_pipeline.cpp

```diff
@@ -9,7 +9,7 @@
 // Or, any cropped not-stolen images will work
 
 // this image should be located at a relative path to the CMake build dir
-const std::string imagePath = "mock_image.jpg";
+const std::string imagePath = "../tests/integration/images/black_mountain/1718243323.jpg";
 
 const std::string refImagePath0 = "../bin/test/test/000000910.jpg";
```

Comment:
These all also need to be fixed.
```suggestion
const std::string refImagePath0 = "../tests/integration/images/matching_cropped/test/000000910.jpg";
```

Reply:
Changed to an array of images at the correct path that gdown downloads them to with `ninja pull_matching_test_images`

https://github.com/tritonuas/obcpp/blob/4287233b77214752e9b35ad875f1bf74d560a29e/tests/integration/cv_pipeline.cpp#L19-L47